examples/konk.yaml

```diff
@@ -2,4 +2,8 @@ apiVersion: konk.infoblox.com/v1alpha1
 kind: Konk
 metadata:
   name: example-konk
-spec: {}
+spec:
+  etcd:
+    auth:
+      client:
+        existingSecret: example-konk-etcd-cert
```

Comment:
Hard to use this generally since the CR creator needs to know the etcd secret name